maint_tools/test_docstrings.py

```diff
@@ -23,6 +23,11 @@
     "RidgeClassifier.fit",
     "RidgeClassifierCV.decision_function",
     "SGDClassifier.decision_function",
+    "VotingRegressor$",
```

Comment:
I am unfamiliar with this `$` notation. I see it above for `LogisticRegression$` and not below for `KernelDensity`. What does it mean and what made you decide to put `$`?

Reply:
It means end of the string with regular expression. LogisticRegression without `$` would match any of its methods, not just the main docstring